docs/manifest.rst

```diff
@@ -1,122 +1,86 @@
 .. _manifest:
+
 Manifest File
 =============
 
-All resource containers have a package.json file which looks like this:
-
-.. code-block:: json
-
-    {
-      "package_version": "0.2",
-      "container_type": "book",
-      "container_slug": "en_ulb_gen_book",
-      "modified_at": 20151222120130,
-      "content_mime_type": "text/usfm",
-      "versification": "kjv",
-
-      "language": {
-        "ietf": "en",
-        "name": "English",
-        "dir": "ltr"
-      },
-
-      "project": {
-        "slug": "gen",
-        "name": "Genesis",
-        "desc": "",
-        "icon": "",
-        "sort": 1,
-        "categories": [
-          "bible-ot"
-        ]
-      },
-
-      "resource": {
-        "slug": "ulb",
-        "name": "Unlocked Literal Bible",
-        "translate_mode": "all",
-        "status": {
-          "checking_entity": [
-            "Wycliffe Associates"
-          ],
-          "checking_level": "3",
-          "version": "3",
-          "comments": "",
-          "contributors": [
-            "Wycliffe Associates"
-          ],
-          "pub_date": "2015-12-17",
-          "license": "CC BY-SA 4.0",
-          "checks_performed": [
-            "keyword",
-            "metaphor"
-          ],
-          "source_translations": [
-            {
-              "container_slug": "en_asv_gen",
-              "version": "1901"
-            }
-          ]
-        },
-      },
-
-      "chunk_status": {
-        "00-title": "draft",
-        "01-01": "reviewed",
-        "01-02": "draft2",
-        "01-title": "draft2",
-        "01-reference": "draft2",
-        "02-01": "reviewed"
-      }
-    }
-
+.. include:: includes/note_keys_required.txt
+
+Resource Containers (RCs) have a manifest.yml that describes it's content and structure.
+Most of the information adheres to the `Dublin Core Meta Data Initiative <http://dublincore.org/documents/dcmi-terms>`_
+and can be found nested within the ``dublin_core`` key.
+
+.. code-block:: yaml
+
+    ---
+
+    dublin_core:
+        type: 'book'
+        conformsto: 'rc0.2'
+        format: 'text/usfm'
+        identifier: 'en-ulb'
+        title: 'Unlocked Literal Bible'
+        subject: 'Bible translation'
+        description: 'The Unlocked Literal Bible is an open-licensed version of the Bible that is intended to provide a form-centric translation of the Bible.'
+        language:
+            identifier: 'en'
+            title: 'English'
+            direction: 'ltr'
+        source:
+        -
+            language: 'en'
+            identifier: 'en-asv'
+            version: '1901'
+        rights: 'CC BY-SA 4.0'
+        creator: 'Wycliffe Associates'
+        contributor:
+            - 'NamesOfContributors'
```

Comment:
Let's put two entries in here just so the example is clear that it's a list and not a string with multiple names.